ui-v2/app/routes/dc/acls/tokens/index.js

```diff
@@ -13,26 +13,24 @@ export default Route.extend(WithTokenActions, {
     },
   },
   beforeModel: function(transition) {
-    return get(this, 'settings')
-      .findBySlug('token')
-      .then(token => {
-        // If you have a token set with AccessorID set to null (legacy mode)
-        // then rewrite to the old acls
-        if (token && get(token, 'AccessorID') === null) {
-          // If you return here, you get a TransitionAborted error in the tests only
-          // everything works fine either way checking things manually
-          this.replaceWith('dc.acls');
-        }
-      });
+    return this.settings.findBySlug('token').then(token => {
+      // If you have a token set with AccessorID set to null (legacy mode)
+      // then rewrite to the old acls
+      if (token && get(token, 'AccessorID') === null) {
```

Comment:
This `get` could be converted too, right?

Reply:
Hmm yeah not sure, I wonder why the codemod skipped this?

This is a `null` check, not an undefined/falsey check, maybe thats why?

Thanks for picking up on this, I'll have a little look and see if I can understand why the codemod skipped it. I'm a bit cautious about just changing it without understanding why it was skipped, I would assume theres a reason behind it? I don't suppose you can think of any other reasons it might skip this?

Anyway will come back here once I've had a little look into it

Reply:
I once wrote a codemod and when things appear in different contexts, they sometimes have to be specifically enumerated. So maybe because this is in an expression…? I’m not sure, but I can’t see any reason this can’t be converted, I think it’s more an instance of how difficult it is to cover all the cases.

Reply:
Hey @backspace 

I've just taken a bit of a closer look through the source code and there are (at a rough estimate) over 100 places where the codemod has left in the `get`. I think some of them are valid (as usage for a safe deep property accessor), but others probably could be switched out.

Right now I've decided I'm a little reluctant to change them in this PR, reasons being:

1. `get` is still perfectly valid to use (although admittedly I'd rather not)
2. There is still a fair amount of work/approval to get:
  - https://github.com/hashicorp/consul/pull/6317
  - https://github.com/hashicorp/consul/pull/5637
  - https://github.com/hashicorp/consul/pull/6285
.. finally merged into `master`. I'd rather leave this for another day and concentrate on getting the rest over the line.
3. I'm trying to avoid changing any behaviour without realising - ie. I want to get the upgrade done with as little alterations as possible (there are already a fair few! 😄 ), right now tests pass, and the application looks good.
4. I'd like to understand why certain `get`s have been left before replacing them.

TL;DR I'd like to convert all these `get`s just not in this PR 😄 

Lemme know what you think.

Reply:
yes, I’m definitely in favour of minimal-intervention Ember version updates, so makes sense to me!